VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/models/panel-model-base.ts

```diff
@@ -160,7 +162,7 @@ export abstract class PanelModel {
             return 2;
         } else if (this.role === Role.QuickLinkParticipant) {
             return 4;
```

Comment:
Maybe consider an alternative to these magi numbers .

Reply:
I would like to but I think that would require too big of a refactor for this. Unfortunately ordering of participants is handled at least 3 different ways in 3 different places. There is a ticket for ordering them consistently that I think this would fall under.